sdk/identity/azure-identity-broker/src/samples/java/com/azure/identity/broker/JavaDocCodeSnippets.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.broker;
+
+import com.azure.identity.InteractiveBrowserCredential;
+import com.sun.jna.Pointer;
+import com.sun.jna.platform.win32.User32;
+import com.sun.jna.platform.win32.WinDef;
+import javafx.stage.Stage;
+
+public class JavaDocCodeSnippets {
+    public void interactiveBrowserBrokerCredentialBuilderSnippet() {
+        // BEGIN: com.azure.identity.broker.interactivebrowserbrokercredentialbuilder.construct
+        InteractiveBrowserBrokerCredentialBuilder builder = new InteractiveBrowserBrokerCredentialBuilder();
+        InteractiveBrowserCredential credential = builder.build();
+        // END: com.azure.identity.broker.interactivebrowserbrokercredentialbuilder.construct
+    }
+
+
+    // BEGIN: com.azure.identity.broker.interactivebrowserbrokercredentialbuilder.getwindowhandle.javafx
+    public long getWindowHandle(Stage stage) {
+        try {
+            WinDef.HWND hwnd = User32.INSTANCE.FindWindow(null, stage.getTitle());
+            return Pointer.nativeValue(hwnd.getPointer());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
This sample, silences the exception, is that what we expect users to do, or it should break the application ?
Adding a comment here may help users to do the right thing.